plugins/plugin-nodejs-debugger/che-plugin-nodejs-debugger-server/src/main/java/org/eclipse/che/plugin/nodejsdbg/server/NodeJsDebugger.java

```diff
@@ -115,7 +116,7 @@ public void addBreakpoint(Breakpoint breakpoint) throws DebuggerException {
             Location location = breakpoint.getLocation();
             pendingBreakpoints.add(location);
             library.setBreakpoint(location.getTarget(), location.getLineNumber());
-            checkActivatedBreakpoints();
+            debuggerCallback.onEvent(new BreakpointActivatedEventImpl(breakpoint));
```

Comment:
I think isn't good idea, because we used checkActivatedBreakpoints method in separate places(in stepOver, stepInto and stepOut methods):

https://github.com/eclipse/che/blob/ccb82196d35534027474a5af37039df771c0349a/plugins/plugin-nodejs-debugger/che-plugin-nodejs-debugger-server/src/main/java/org/eclipse/che/plugin/nodejsdbg/server/NodeJsDebugger.java#L222
https://github.com/eclipse/che/blob/ccb82196d35534027474a5af37039df771c0349a/plugins/plugin-nodejs-debugger/che-plugin-nodejs-debugger-server/src/main/java/org/eclipse/che/plugin/nodejsdbg/server/NodeJsDebugger.java#L209
https://github.com/eclipse/che/blob/ccb82196d35534027474a5af37039df771c0349a/plugins/plugin-nodejs-debugger/che-plugin-nodejs-debugger-server/src/main/java/org/eclipse/che/plugin/nodejsdbg/server/NodeJsDebugger.java#L196

I gues it make sense to used the checkActivatedBreakpoints method in case when you jump from one file to another.  
@tolusha  WDYT?